tox.ini

```diff
@@ -11,16 +11,26 @@ deps =
     -rrequirements-test.txt
 whitelist_externals =
     make
+    mkdir
+    rm
 passenv = TOX_ENV_DIR
 setenv =
 	CACHITO_TESTING=true
+    PROMETHEUS_MULTIPROC_TEMP_DIR={envtmpdir}/prometheus_metrics
+
 usedevelop = true
+commands_pre =
+    mkdir -p {envtmpdir}/prometheus_metrics
```

Comment:
Same as the dockerfile, shouldn't Cachito create this automatically?

Reply:
Yes.  Removed.

Reply:
Did you include the `tox.ini` changes? I still see this here, and the empty line at the end

Reply:
I missed it yesterday in the process of fixing the issue with the multiprocess bit.  Fixed.